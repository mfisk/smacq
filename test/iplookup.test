#!/bin/sh

${BINDIR}smacqq -g -m <<EOM
pcapfile $TEST/0325@1112-snort.pcap | iplookup srcip 192.168.0.0/24 | const -f label shadownet | print label srcip dstip 
pcapfile $TEST/0325@1112-snort.pcap | iplookup srcip 192.168.0.54/32 | const -f label foohost | print label srcip dstip 
EOM
