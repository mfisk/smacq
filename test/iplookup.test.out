foohost	192.168.0.54	192.168.0.210
foohost	192.168.0.54	192.168.0.210
foohost	192.168.0.54	192.168.0.210
foohost	192.168.0.54	192.168.0.210
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
shadownet	192.168.0.200	192.168.0.54
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
foohost	192.168.0.54	192.168.0.200
